### PR TITLE
[P2022-183] Basic modal

### PR DESCRIPTION
**Pull request by jprymak**

### CODE REVIEW

File: src/components/BasicModal/BasicModal.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
```
Comment (Voland91) [inline src/components/BasicModal/BasicModal.tsx:1]: Is this React import necessary? It's nothing wrong, I'm just curious :)

**Review by Voland91 — APPROVED** [review]
Looks good :)

File: src/components/BasicModal/BasicModal.tsx (hunk 1 above)
Comment (jprymak) [reply src/components/BasicModal/BasicModal.tsx:1]: image

Looks like it is in this case :)

Comment (jprymak) [reply src/components/BasicModal/BasicModal.tsx:1]: Just how it was in the mui code snippet. I usually use {} for React imports

**Pull request merged**